.github/workflows/ci.yml

```diff
@@ -12,22 +12,15 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Set up Go 1.20
-        uses: actions/setup-go@v4
-        with:
-          go-version: '1.20'
-        id: go
-
       - name: Check out code into the Go module directory
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
-      - name: Restore cache
-        uses: actions/cache@v3
+      - name: Set up Go
+        uses: actions/setup-go@v5
         with:
-          path: ~/go/pkg/mod
-          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-go-
+          go-version: '1.23'
```

Comment:
```suggestion
          go-version: '1.23.3'
```

Reply:
I prefer `1.23` because it refers to the latest minor version. For example, when `1.23.4` becomes available, we don't need to upgrade the minor version here.

Reply:
How about specifying stable ?